program/src/instruction.rs

```diff
@@ -421,6 +421,49 @@ pub enum StakeInstruction {
         desc = "System program"
     )]
     DistributeRewards(u64),
+
+    /// Initializes stake account data for a SOL staker.
+    ///
+    /// NOTE: Anybody can create the stake account for a SOL staker. For new
+    /// accounts, the authority is initialized to the stake state account's withdrawer.
+    #[account(
+        0,
+        name = "config",
+        desc = "Stake config account"
+    )]
+    #[account(
+        1,
+        writable,
+        name = "sol_staker_stake",
+        desc = "SOL staker stake account (pda of `['stake::state::sol_staker_stake', stake state, config]`)"
+    )]
+    #[account(
+        2,
+        writable,
+        name = "validator_stake",
+        desc = "Validator stake account (pda of `['stake::state::validator_stake', validator, config]`)"
+    )]
+    #[account(
+        3,
+        name = "stake_state",
+        desc = "SOL stake state account"
```

Comment:
nit: call it a SOL stake?
```suggestion
        name = "sol_stake",
        desc = "SOL stake account"
```

Reply:
Should we also rename the field on the `SolStakerStake`? https://github.com/paladin-bladesmith/stake-program/blob/main/program/src/state/sol_staker_stake.rs#L40

Reply:
Ah yeah, please, maybe `sol_stake` or `sol_stake_address`? I'd lean towards the first one since we don't use `_address` anywhere else